bitcoin/build.rs

```diff
@@ -24,10 +24,12 @@ fn main() {
         .parse::<u64>()
         .expect("invalid Rust minor version");
 
-    let msrv = std::env::var_os("CARGO_PKG_RUST_VERSION").unwrap();
-    let mut msrv_semvar = msrv.to_str().unwrap().split(".");
-    msrv_semvar.next().unwrap(); //skip major version number
-    let msrv_minor = msrv_semvar.next().unwrap().parse::<u64>().unwrap();
+    let msrv = std::env::var("CARGO_PKG_RUST_VERSION").unwrap();
+    let msrv = msrv.to_string();
```

Comment:
It's already a string - that was the point of my request.

Reply:
good point.  fixed.